Config.sample.toml

```diff
@@ -0,0 +1,215 @@
+# ========== Configuration file for AutoMaxLair ==========
+# What follows is the set of configurations for the AutoMaxLair program.
+# Please be sure to read carefully the descriptions of the settings and
+# adjust them as necessary for your circumstances.
+#
+# PLEASE NOTE that you will need to review the settings after each run.
+#
+# TOML also has specific formatting (useful so that the program runs properly!)
+# so please be sure to not add or remove decimal points or quotes
+# and do not modify the style in this file. Also, `true` and `false`
+# should ALWAYS be lower case!
+# ========================================================
+
+# === BOSS ===
+# This is the boss that you are resetting for. This is a string for the name.
+#   NOTE: this allows the program to properly select Pokémon and Moves throughout the run.
+#   IMPORTANT: Use entirely lower case letters and replace all spaces with hyphens.
+#       e.g. "moltres" "kyurem" "tapu-koko" "tapu-lele"
+#   The boss name also needs to be its English name due to internal storage.
+#   Bosses with multiple formes need to have the forme specified.
+#   This includes the following:
+#       "giratina-altered", "thundurus-incarnate", "tornadus-incarnate", "landorus-incarnate", "zygarde-50"
+BOSS = "dialga"
+# === BASE_BALL ===
+# This is the Pokéball that should be used for all Pokémon except for the boss.
+#   This can be a string for the ball name, or it can be DEFAULT to always choose
+#   the first ball in the catch menu. Enter the name as seen in your bag.
+BASE_BALL = "DEFAULT"
+# === BASE_BALLS ===
+# This is the current quantity of the ball that you selected in BASE_BALL
+BASE_BALLS = 600
+# === LEGENDARY_BALL ===
+# This is the Pokéball that should be used for just the legendary boss Pokémon.
+#   This can be a string for the ball name, or it can be DEFAULT to always choose 
+#   the first ball in the catch menu. Enter the name as seen in your bag.
+LEGENDARY_BALL = "Premier Ball"
```

Comment:
We could maybe switch to "Premier" here. "Ball" doesn't add any selectivity and could result in false negatives.

Reply:
That's true, might be a good call. I can do that.